### PR TITLE
enh(registrer): Add registration of servers

### PR DESCRIPTION
**Pull request by lpinsivy**

### CODE REVIEW

File: en/reporting/installation.md [hunk 1]
```diff
@@ -453,3 +453,78 @@ Restart the service cron::
...
 
 Centreon MBI installation is now complete, check [the tutorial](../getting-started/analyze-resources-availability.html)
+
+#### Register the server
+
+To register your Centreon MBI server to the Centreon Central server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <REMOTE_SERVER_NAME>
```
Comment (loiclau) [inline en/reporting/installation.md:463]: -t mbi

Comment (lpinsivy) [reply en/reporting/installation.md]: GG, I made the same for map...

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 2]
```diff
@@ -123,6 +123,106 @@ Then restart your server with the following command:
...
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
+
+> You credetials are incorrect for the **<API_ACCOUNT>**.
```
Comment (sc979) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:208]:
```suggestion
> Your credentials are incorrect for the **<API_ACCOUNT>**.
```

File: en/installation/installation-of-a-poller/using-packages.md [hunk 3]
```diff
@@ -68,6 +83,106 @@ systemctl start centreontrapd snmptrapd
...
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
+
+> You credetials are incorrect for the **<API_ACCOUNT>**.
```
Comment (sc979) [inline en/installation/installation-of-a-poller/using-packages.md:168]:
```suggestion
> Your credentials are incorrect for the **<API_ACCOUNT>**.
```

File: en/installation/installation-of-a-remote-server/using-centreon-iso.md [hunk 4]
```diff
@@ -141,65 +141,106 @@ Conclude installation by performing [web installation steps](../web-and-post-ins
...
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
+
+> You credetials are incorrect for the **<API_ACCOUNT>**.
```
Comment (sc979) [inline en/installation/installation-of-a-remote-server/using-centreon-iso.md:226]:
```suggestion
> Your credentials are incorrect for the **<API_ACCOUNT>**.
```

File: en/installation/installation-of-a-remote-server/using-packages.md [hunk 5]
```diff
@@ -193,65 +193,106 @@ Conclude installation by performing
...
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
+
+> You credetials are incorrect for the **<API_ACCOUNT>**.
```
Comment (sc979) [inline en/installation/installation-of-a-remote-server/using-packages.md:278]:
```suggestion
> Your credentials are incorrect for the **<API_ACCOUNT>**.
```

File: en/reporting/installation.md [hunk 6]
```diff
@@ -453,3 +453,78 @@ Restart the service cron::
...
+parent server address: 192.168.0.1
+
+Do you want to register this server with those informations ? (y/n)y
+```
+
+You will receive the validation of the Centreon central or the Remote Server server:
+
+``` shell
+message: The 'centreon-mbi-server' Platform: 'centreon-mbi-server@192.168.0.2.2' linked to '192.168.0.1' has been added
+```
```
Comment (sc979) [inline en/reporting/installation.md:530]:
```suggestion
```

**Review by sc979 — CHANGES_REQUESTED** [review]
MBI registering is not achieved

File: en/installation/installation-of-a-poller/using-packages.md [hunk 7]
```diff
@@ -68,6 +83,106 @@ systemctl start centreontrapd snmptrapd
 > Active monitoring service will be started following the generation of its
 > configuration.
 
+## Register the server
+
+To register it to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <REMOTE_SERVER_NAME>
```
Comment (quanghungb) [inline en/installation/installation-of-a-poller/using-packages.md:92]: -n <REMOTE_SERVER_NAME>

here we are registering a Poller, so this is a Poller name ?

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 8]
```diff
@@ -123,6 +123,106 @@ Then restart your server with the following command:
...
+```
+
+Example:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u admin -t Poller -h 192.168.0.1 -n poller-1
+```
+
+> Replace **<IP_CENTREON_CENTRAL>** by the IP of the Centreon server seen by the poller or by the Remote Server if you
+> want to link your server to it..
```
Comment (sc979) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:142]: I can't find where the software collection repository installation is explained for a poller
As php-cli-72 is required

File: en/graph-views/install.md [hunk 9]
```diff
@@ -262,6 +262,93 @@ Enable the service to start up automatically on server boot:
...
+
+```shell
+yum install -y centos-release-scl
+```
+
+To register your Centreon Map server to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t map -h <IP_CENTREON_CENTRAL> -n <SERVER_NAME>
```
Comment (jeremyjaouen) [inline en/graph-views/install.md:275]:
```suggestion
-t map -h <IP_TARGET_NODE> -n <SERVER_NAME>
```
The IP could be a Central or Remote

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 10]
```diff
@@ -123,6 +123,119 @@ Then restart your server with the following command:
...
+
+```shell
+yum install -y centos-release-scl
+```
+
+To register it to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <POLLER_NAME>
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:138]:
```suggestion
-t Poller -h <IP_TARGET_NODE> -n <POLLER_NAME>
```

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 11]
```diff
@@ -123,6 +123,119 @@ Then restart your server with the following command:
...
+``` shell
+2020-10-16T17:19:37+02:00 [INFO]: The CURRENT NODE 'poller': 'poller-1@192.168.0.2' linked to TARGET NODE: '192.168.0.1' has been added
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:219]: This section isn't up to date

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 12]
```diff
@@ -123,6 +123,119 @@ Then restart your server with the following command:
...
+
+> Your credentials are incorrect for the **<API_ACCOUNT>**.
+
+``` shell
+error code: 500
+error message: Access Denied.
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:226]: This section isn't up to date

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 13]
```diff
@@ -123,6 +123,119 @@ Then restart your server with the following command:
...
+
+> The **<API_ACCOUNT>** doesn't have access to configuration API.
+
+``` shell
+error code: 400
+error message: Can't connect to the api
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:233]: This section isn't up to date

File: en/installation/installation-of-a-poller/using-packages.md [hunk 14]
```diff
@@ -68,6 +83,119 @@ systemctl start centreontrapd snmptrapd
...
+
+```shell
+yum install -y centos-release-scl
+```
+
+To register it to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <POLLER_NAME>
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-packages.md:98]:
```suggestion
-t Poller -h <IP_TARGET_NODE> -n <POLLER_NAME>
```

File: en/installation/installation-of-a-poller/using-packages.md [hunk 15]
```diff
@@ -68,6 +83,119 @@ systemctl start centreontrapd snmptrapd
...
+``` shell
+2020-10-16T17:19:37+02:00 [INFO]: The CURRENT NODE 'poller': 'poller-1@192.168.0.2' linked to TARGET NODE: '192.168.0.1' has been added
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-packages.md:179]: This section isn't up to date

File: en/installation/installation-of-a-poller/using-packages.md [hunk 16]
```diff
@@ -68,6 +83,119 @@ systemctl start centreontrapd snmptrapd
...
+
+> Your credentials are incorrect for the **<API_ACCOUNT>**.
+
+``` shell
+error code: 500
+error message: Access Denied.
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-packages.md:186]: This section isn't up to date

File: en/installation/installation-of-a-poller/using-packages.md [hunk 17]
```diff
@@ -68,6 +83,119 @@ systemctl start centreontrapd snmptrapd
...
+
+> The **<API_ACCOUNT>** doesn't have access to configuration API.
+
+``` shell
+error code: 400
+error message: Can't connect to the api
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-poller/using-packages.md:193]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-centreon-iso.md [hunk 18]
```diff
@@ -141,65 +141,113 @@ Conclude installation by performing [web installation steps](../web-and-post-ins
...
+``` shell
+2020-10-16T17:19:37+02:00 [INFO]: The CURRENT NODE 'remote': 'remote-1@192.168.0.2' linked to TARGET NODE: '192.168.0.1' has been added
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-centreon-iso.md:231]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-centreon-iso.md [hunk 19]
```diff
@@ -141,65 +141,113 @@ Conclude installation by performing [web installation steps](../web-and-post-ins
...
+
+> Your credentials are incorrect for the **<API_ACCOUNT>**.
+
+``` shell
+error code: 500
+error message: Access Denied.
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-centreon-iso.md:238]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-centreon-iso.md [hunk 20]
```diff
@@ -141,65 +141,113 @@ Conclude installation by performing [web installation steps](../web-and-post-ins
...
+
+> The **<API_ACCOUNT>** doesn't have access to configuration API.
+
+``` shell
+error code: 400
+error message: Can't connect to the api
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-centreon-iso.md:245]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-packages.md [hunk 21]
```diff
@@ -193,65 +193,113 @@ Conclude installation by performing
...
+``` shell
+2020-10-16T17:19:37+02:00 [INFO]: The CURRENT NODE 'remote': 'remote-1@192.168.0.2' linked to TARGET NODE: '192.168.0.1' has been added
+```
+
+### Main errors messages
+
+``` shell
+error code: 401
+error message: Invalid credentials
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-packages.md:283]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-packages.md [hunk 22]
```diff
@@ -193,65 +193,113 @@ Conclude installation by performing
...
+
+> Your credentials are incorrect for the **<API_ACCOUNT>**.
+
+``` shell
+error code: 500
+error message: Access Denied.
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-packages.md:290]: This section isn't up to date

File: en/installation/installation-of-a-remote-server/using-packages.md [hunk 23]
```diff
@@ -193,65 +193,113 @@ Conclude installation by performing
...
+
+> The **<API_ACCOUNT>** doesn't have access to configuration API.
+
+``` shell
+error code: 400
+error message: Can't connect to the api
+```
```
Comment (jeremyjaouen) [inline en/installation/installation-of-a-remote-server/using-packages.md:297]: This section isn't up to date

File: fr/graph-views/install.md [hunk 24]
```diff
@@ -262,8 +262,93 @@ Enable the service to start up automatically on server boot:
...
+
+```shell
+yum install -y centos-release-scl
+```
+
+To register your Centreon Map server to the Centreon Central server or a Remote server, execute the following command:
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t map -h <IP_CENTREON_CENTRAL> -n <SERVER_NAME>
```
Comment (jeremyjaouen) [inline fr/graph-views/install.md:275]:
```suggestion
-t map -h <IP_TARGET_NODE> -n <SERVER_NAME>
```

File: fr/graph-views/install.md [hunk 25]
```diff
@@ -262,8 +262,93 @@ Enable the service to start up automatically on server boot:
 
     systemctl enable centreon-map
 
-Centreon Map server is now started and enabled, let's install the interface part of the extension.
+Install the Software Collections repository using this command:
```
Comment (jeremyjaouen) [inline fr/graph-views/install.md:265]: All this doc shouldn't be in french ?

File: fr/installation/installation-of-a-poller/using-centreon-iso.md [hunk 26]
```diff
@@ -121,6 +121,95 @@ Redémarrez votre système avec la commande:
...
+
+```shell
+yum install -y centos-release-scl
+```
+
+Pour l'enregistrer sur le serveur Centreon Central ou un serveur distant, exécutez la commande suivante :
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <POLLER_NAME>
```
Comment (jeremyjaouen) [inline fr/installation/installation-of-a-poller/using-centreon-iso.md:136]:
```suggestion
-t Poller -h <IP_TARGET_NODE> -n <POLLER_NAME>
```

File: fr/installation/installation-of-a-poller/using-packages.md [hunk 27]
```diff
@@ -71,6 +84,89 @@ systemctl start centreontrapd snmptrapd
 > Le service de supervision active sera démarré suite à la génération de sa
 > configuration.
 
+## Enregistrer le serveur
+
+Pour l'enregistrer sur le serveur Centreon Central ou un serveur distant, exécutez la commande suivante :
+
+``` shell
+/opt/rh/rh-php72/root/bin/php /usr/share/centreon/bin/registerServerTopology.php -u <API_ACCOUNT> \
+-t Poller -h <IP_CENTREON_CENTRAL> -n <POLLER_NAME>
```
Comment (jeremyjaouen) [inline fr/installation/installation-of-a-poller/using-packages.md:93]:
```suggestion
-t Poller -h <IP_TARGET_NODE> -n <POLLER_NAME>
```

**Review by jeremyjaouen — CHANGES_REQUESTED** [review]

File: fr/graph-views/install.md (hunk 25 above)
Comment (lpinsivy) [reply fr/graph-views/install.md:265]: Yes but we need time to translate ;)

**Review by Fxdpt — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**